app/models/event_schedule.rb

```diff
@@ -70,6 +70,14 @@ def happening_now?(threshold = 30.minutes)
     event_time_range.overlaps?(now_range)
   end
 
+  def happening_next?
```

Comment:
Let's call this `happening_later?` since we're using next to mean just at the next time.